data/report_templates/project_summary.md

```diff
@@ -181,10 +181,29 @@ reverse reads (if the run was a paired-end run).
 
 The naming of the files follow the convention:
 
+{% if not project.is_hiseqx -%}
 ```
 [LANE]_[DATE]_[POSITION][FLOWCELL]_[NGI-NAME]_[READ].fastq.gz
 ```
 
+* _LANE:_ Lane number the fastq file(sample) was sequenced
+* _DATE:_ Date of sequencing
+* _POSITION:_ Position of the flowcell in the sequencer
+* _FLOWCELL:_ Unique flowcell indentifier
+* _NGI-NAME:_ Internal NGI sample indentifier
+* _READ:_ Forward(1) or reverese(2) read indentifier
+{%- else -%}
+```
+[NGI-NAME]_[BCL-CONVERSION-ID]_[LANE]_[READ]_[VOLUME].fastq.gz
+```
+
+* _NGI-NAME:_ Internal NGI sample indentifier
+* _BCL-CONVERSION-ID:_ Indentifier set by bcl2fastq tool while demultiplexing
+* _LANE:_ Lane number the fastq file(sample) was sequenced
+* _READ:_ Forward(1) or reverese(2) read indentifier
+* _VOLUME:_ volume number of fastq file, always 001 (except for very huge files splitted into volumes)
```

Comment:
_"Volume index when file is large enough to be split into volumes"_

Though to be honest, if it's (basically) always `_001`, maybe just remove this line and replace `_[VOLUME]` up above with `_001`?